app/doc/responses.go

```diff
@@ -121,3 +121,44 @@ type groupsGetGroupProgressResponse struct {
 		AvgTimeSpent float32 `json:"avg_time_spent"`
 	}
 }
+
+// OK. Success response with teams progress on items
+// swagger:response groupsGetTeamProgressResponse
+type groupsGetTeamProgressResponse struct {
+	// in: body
+	Body []struct {
+		// The team’s `group_id`
+		// required:true
+		GroupID int64 `json:"group_id,string"`
+		// required:true
+		ItemID int64 `json:"item_id,string"`
+		// The best score across all attempts, or 0
+		// required:true
+		Score float32 `json:"score"`
+		// Max of `bValidated` across all attempts, or `false`
+		// required:true
+		Validated bool `json:"validated"`
+		// Max across all attempts of the team (`null` if no attempts)
```

Comment:
I think the description can be removed. And you have to add `Nullable`

Reply:
fixed